.storybook/themes/commerce.scss

```diff
@@ -234,6 +234,41 @@
 				),
 			),
 			blocks: (
+				category-carousel: (
+					components: (
+						carousel: (
+							--slides: 1,
+						),
+						carousel-button: (
+							border-radius: var(--global-spacing-1),
+							background-color: var(--global-neutral-1),
+							padding: 0.75em,
+						),
+						carousel-track: (
+							gap: var(--global-spacing-5),
+						),
+						carousel-actions: (
+							display: none,
+							padding-left: var(--global-spacing-3),
+							padding-right: var(--global-spacing-3),
+							max-width: 95.8%,
+							// (48 + 32 + 1220 + 32 + 48) / 1440
+						),
+						heading: (
+							--heading-truncation: 2,
+							font-size: var(--global-font-size-11),
+						),
+						icon: (
+							height: var(--global-font-size-9),
+							width: var(--global-font-size-9),
+						),
+					),
+				),
+				category-carousel-slide-title: (
+					margin-left: initial,
```

Comment:
curious what's happening here. do you need the initial? is this a hold-over from when we needed to redeclare across breakpoints?

Reply:
This is an effect of how I did the them styling.  I suppose I could name the carousel header, but because I themed the header components of this block to have a margin/max-width, this resets the card headers.

Reply:
I have created the category-carousel-title token and placed that styling there to make this clearer.